aderyn_core/src/fscloc/insight.rs

```diff
@@ -34,21 +35,24 @@ impl From<&TokenDescriptor> for TokenInsight {
 
         // if value.token_type != TokenType::MultilineComment
         //     && value.token_type != TokenType::SinglelineComment
-        // && insight.code_lines.actual_first_line != -1
-        //{
-        // print!(
-        //     "{:?}, {}-{}\n",
-        //     value.token_type.clone(),
-        //     value.start_line,
-        //     value.end_line
-        // );
-        // println!("{}", value.content.clone());
+        //     && insight.start_line == 3
```

Comment:
The  statements helped me a lot with intuition on how I had organized these things and that's why I was able to solve this problem. Is there anything else we can do to avoid removing it from the codebase? 

Surely I can bookmark this commit to see these comments along with where they were .. but just asking for alternative advice ..

Reply:
We can leave in here if you feel it's really necessary.

Do these lines provide context that debugging cannot?
Is there context we can add as comments in the function description docblock?